File: PRACTICA 1/punto.java

// clase Punto
public class punto {
    public float x;
    public float y;
    
public punto(float x, float y){
this.x = x;
this.y = y;
}

public float[] coord_cartesianas() {
 return new float[] {this.x, this.y};
}
public float[] coord_polares() {
 float radio = (float)Math.sqrt(this.x * this.x + this.y * this.y);
 float angulo = (float) Math.atan(this.y/this.y);
 angulo = (float) Math.toDegrees(angulo);
 return new float[] {radio, angulo};
}
public String toString(){
return String.format("(%.2f, %.2f)", this.x, this.y);
}


    
}
// clase Principal 
public class Tarea1 {

    public static void main(String[] args) {
        // TODO code application logic here
        punto p1 = new punto(0,3);
System.out.println(p1);
float a[] = p1.coord_cartesianas();
System.out.println(a[0] + " " + a[1]);
float b[] = p1.coord_polares();
System.out.println(b[0] + " " + b[1]);
    }
    
}

File: PRACTICA 1/punto.py
import math

class punto:
    def __init__(self,x,y):
        self.x = x
        self.y = y

    def coord_cartesianas(self):
        return (self.x , self.y)

    def coord_polares(self):
        radio = math.sqrt(self.x * self.x + self.y * self.y)
        angulo = math.sqrt(self.y/self.y)
        angulo = math.degrees(angulo)
        return radio, angulo

    def __str__(self):
        return "({:.2f},{:.3f})" .format(self.x, self.y)

p = punto(5,7)
print(p)
x,y = p.coord_cartesianas()
print(x,y)
r,a = p.coord_polares()
print(r,a)